test/e2e/validate/config.go

```diff
@@ -57,126 +59,207 @@ var ValidateTests = []struct {
+		ResourceRaw:       kyvernoPodWithSmallImage,
 		MustSucceed:       true,
 	},
 	{
 		// Case for small image validation
 		TestDescription:   "checks that images are small",
 		PolicyName:        "check-large-images",
-		PolicyRaw:         kyverno_small_image_policy,
+		PolicyRaw:         kyvernoSmallImagePolicy,
 		ResourceName:      "pod-with-large-image",
 		ResourceNamespace: "test-validate",
 		ResourceGVR:       podGVR,
-		ResourceRaw:       kyverno_pod_with_large_image,
+		ResourceRaw:       kyvernoPodWithLargeImage,
 		MustSucceed:       false,
 	},
 	{
 		// Case for yaml signing validation
 		TestDescription:   "checks that unsigned yaml manifest is blocked",
 		PolicyName:        "check-yaml-signing",
-		PolicyRaw:         kyverno_yaml_signing_validate_policy,
+		PolicyRaw:         kyvernoYamlSigningValidatePolicy,
 		ResourceName:      "test-deployment",
 		ResourceNamespace: "test-validate",
 		ResourceGVR:       deploymentGVR,
-		ResourceRaw:       kyverno_yaml_signing_validate_resource_1,
+		ResourceRaw:       kyvernoYamlSigningValidateResource1,
 		MustSucceed:       false,
 	},
 	{
 		// Case for yaml signing validation
 		TestDescription:   "checks that signed yaml manifest is created",
 		PolicyName:        "check-yaml-signing",
-		PolicyRaw:         kyverno_yaml_signing_validate_policy,
+		PolicyRaw:         kyvernoYamlSigningValidatePolicy,
 		ResourceName:      "test-deployment",
 		ResourceNamespace: "test-validate",
 		ResourceGVR:       deploymentGVR,
-		ResourceRaw:       kyverno_yaml_signing_validate_resource_2,
+		ResourceRaw:       kyvernoYamlSigningValidateResource2,
 		MustSucceed:       true,
 	},
 	{
 		// Case for failing X.509 certificate decoding validation
 		TestDescription:   "checks if the public key modulus of base64 encoded x.509 certificate is same as the pem x.509 certificate",
 		PolicyName:        "check-x509-decode",
-		PolicyRaw:         kyverno_decode_x509_certificate_policy,
+		PolicyRaw:         kyvernoDecodeX509CertificatePolicy,
 		ResourceName:      "test-configmap",
 		ResourceNamespace: "test-validate",
 		ResourceGVR:       configmapGVR,
-		ResourceRaw:       kyverno_decode_x509_certificate_resource_fail,
+		ResourceRaw:       kyvernoDecodeX509CertificateResourceFail,
 		MustSucceed:       false,
 	},
 	{
 		// Case for passing X.509 certificate decoding validation
 		TestDescription:   "checks if the public key modulus of base64 encoded x.509 certificate is same as the pem x.509 certificate",
 		PolicyName:        "check-x509-decode",
-		PolicyRaw:         kyverno_decode_x509_certificate_policy,
+		PolicyRaw:         kyvernoDecodeX509CertificatePolicy,
 		ResourceName:      "test-configmap",
 		ResourceNamespace: "test-validate",
 		ResourceGVR:       configmapGVR,
-		ResourceRaw:       kyverno_decode_x509_certificate_resource_pass,
+		ResourceRaw:       kyvernoDecodeX509CertificateResourcePass,
 		MustSucceed:       true,
 	},
 }
+
+type SubresourceValidationTest struct {
```

Comment:
Thanks for checking and adding e2e validate tests. Do we need to enable it in the makefile?

Reply:
Enabled.